src/software/convert/main_convertLDRToHDR.cpp

```diff
@@ -0,0 +1,484 @@
+  }
+
+  // set verbose level
+  system::Logger::get()->setLogLevel(verboseLevel);
+
+  // set the correct weight functions corresponding to the string parameter
+  hdr::rgbCurve fusionWeight(channelQuantization);
+  fusionWeight.setFunction(fusionWeightFunction);
+
+  hdr::rgbCurve calibrationWeight(channelQuantization);
+  std::transform(calibrationWeightFunction.begin(), calibrationWeightFunction.end(), calibrationWeightFunction.begin(), ::tolower);
+  if(calibrationWeightFunction == "default")
+  {
+    switch(calibrationMethod)
+    {
+    case ECalibrationMethod::NONE:        break;
+    case ECalibrationMethod::LINEAR:      break;
+    case ECalibrationMethod::DEBEVEC:     calibrationWeight.setTriangular();
+    case ECalibrationMethod::ROBERTSON:   calibrationWeight.setRobertsonWeight();
+    case ECalibrationMethod::GROSSBERG:   calibrationWeight.setGaussian();
+    }
+  }
+  else
+    calibrationWeight.setFunction(hdr::EFunctionType_stringToEnum(calibrationWeightFunction));
+
+  // get all valid images from input
+  std::vector<std::string> inputImagesNames;
+  std::vector<std::string> stemImages;
+  std::vector<std::string> nameImages;
+
+  const std::regex validExtensions("\\.(?:"
+  // Basic image file extensions
+  "jpg|jpeg|png|tiff|tif|"
+  // RAW image file extensions:
+  "3fr|" // Hasselblad
+  "arw|" // Sony
+  "crw|cr2|cr3|" // Canon
+  "dng|" // Adobe
+  "kdc|" // Kodak
+  "mrw|" // Minolta
+  "nef|nrw|" // Nikon
+  "orf|" // Olympus
+  "ptx|pef|" // Pentax
+  "raf|" // Fuji
+  "R3D|" // RED
+  "rw2|" // Panasonic
+  "srw|" // Samsung
+  "x3f" // Sigma
+  ")"
+  );
+
+  for(const std::string& entry: imagesFolder)
+  {
+    if(fs::is_directory(fs::path(entry)))
+    {
+      for(const auto& file: fs::directory_iterator(fs::path(entry)))
+      {
+        std::string extension = file.path().extension().string();
+        std::transform(extension.begin(), extension.end(), extension.begin(), ::tolower);
+        if(fs::is_regular_file(file.status()) && std::regex_match(extension, validExtensions))
+        {
+            inputImagesNames.push_back(file.path().string());
+            stemImages.push_back(file.path().stem().string());
+            nameImages.push_back(file.path().filename().string());
+        }
+      }
+    }
+    else
+    {
+      std::string extension = fs::path(entry).extension().string();
+      std::transform(extension.begin(), extension.end(), extension.begin(), ::tolower);
+      if(fs::is_regular_file(fs::path(entry)) && std::regex_match(extension, validExtensions))
+      {
+        inputImagesNames.push_back(entry);
+        stemImages.push_back(fs::path(entry).stem().string());
+        nameImages.push_back(fs::path(entry).filename().string());
+      }
+    }
+  }
+  if(inputImagesNames.empty())
+  {
+    ALICEVISION_LOG_ERROR("No valid input images. Please give a list of LDR images or a folder path containing them (accepted formats are: .jpg .jpeg .png .tif .tiff .cr2 .rw2");
```

Comment:
ALICEVISION_LOG_ERROR("No valid input images. Please give a list of LDR images or a folder path containing them (accepted formats are: " << validExtensions << ")")